presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -47,6 +47,10 @@ statement
     | CREATE (OR REPLACE)? VIEW qualifiedName AS query                 #createView
     | DROP VIEW (IF EXISTS)? qualifiedName                             #dropView
     | CALL qualifiedName '(' (callArgument (',' callArgument)*)? ')'   #call
+    | GRANT
+        (privilege (',' privilege)* | ALL PRIVILEGES)
+        ON (TABLE)? qualifiedName TO (grantee=identifier | PUBLIC)
```

Comment:
Done in https://github.com/Teradata/presto/commit/f0ccf281bc9f13601cc1826efe262a57016e1675